lib/Cake/Test/Case/View/Helper/HtmlHelperTest.php

```diff
@@ -583,14 +583,15 @@ public function testBase64ImageTag()
 	 * testLoadConfigWrongFile method
 	 *
 	 * @return void
-	 * @expectedException NotFoundException
+	 * @expectedException InvalidArgumentException
 	 */
 
-	public function testBase64NotFoundException()
+	public function testBase64InvalidArgumentException()
 	{
 		$this->Html->request->webroot = '';
 
-		$this->Html->image('non-existent-image.png', ['base64' => true]);
+		$return = $this->Html->image('non-existent-image.png', ['base64' => true]);
+		debug($return);
```

Comment:
Ooops, side effect of trying out work out why I wasn't getting the throw with a straight File::open on a non-existing file (because File::open will create it if required ¬_¬).

This should be resolved.